Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -0,0 +1,1244 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Net.NetworkInformation;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+using Jellyfin.Networking.Configuration;
+using MediaBrowser.Common.Configuration;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using NetworkCollection;
+using NetworkCollection.Udp;
+
+namespace Jellyfin.Networking.Manager
+{
+    /// <summary>
+    /// Class to take care of network interface management.
+    /// </summary>
+    public class NetworkManager : INetworkManager, IDisposable
+    {
+        /// <summary>
+        /// Contains the description of the interface along with its index.
+        /// </summary>
+        private readonly Dictionary<string, int> _interfaceNames;
+
+        /// <summary>
+        /// Threading lock for network interfaces.
```

Comment:
`_interfaceNames` or `_interfaceAddresses`?
Change the name of the field to make this more clear

Reply:
altered comment.